core/tests/protractor_utils/CollectionEditorPage.js

```diff
@@ -159,7 +159,9 @@ var CollectionEditorPage = function() {
     // converted to action.sendKeys(..)
     // because doing so causes the e2e test to fail with the error
     // "Failed: Cannot read property 'bind' of undefined".
```

Comment:
[Future work] This looks like some general binding issue, instead of using a pragma and TODO, I think we should prioritize fixing this to avoid the further copy of this code.

Reply:
fixed